src/main/java/edu/kit/hci/soli/domain/Room.java

```diff
@@ -0,0 +1,22 @@
+package edu.kit.hci.soli.domain;
+
+import edu.kit.hci.soli.controller.BookingsController;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import lombok.Data;
+
+/**
+ * The datamodel for a room as it is stored in the database
+ */
+@Entity
+@Data
+public class Room {
+
+    @Id
+    private Long id;
+
+    @OneToOne
+    private Booking currentBooking;
```

Comment:
IMO this should be part of the bookings repository (or, if you really think it should always be there, of our domain model), not of our database model

Reply:
This seems good now. I can add the repository method later.